api/app/Modules/Audit/Http/Transformers/RevisionTransformer.php

```diff
@@ -39,7 +39,7 @@ public function toArray(Revision $revision): array
             'previous' => $this->prepareSnapshot($revision->old_values, $revision->getEntityType()),
             'snapshot' => $this->prepareSnapshot($revision->new_values, $revision->getEntityType()),
             'meta' => $this->getMeta($revision),
-            'createdAt' => $this->dateTime($revision->created_at),
```

Comment:
Can you check if this property from the model has the correct date cast set up? Looks like the error says it's coming back as a string.

Reply:
Reverted this change back